varlink_derive/src/lib.rs

```diff
@@ -1,9 +1,16 @@
-//! Macro for generating modules from a varlink interface definition
+//! Macros for generating modules from a varlink interface definition
```

Comment:
Nitpick, you split other logical changes into separate commits so well, I feel bad to ask for this but could you please split the "varlink_derive: use path relative to manifest dir" commit a bit more. Feel free to put all typo fixes in the same commit.

Reply:
Thanks for the review! I split all the changes from this and other commits granularly, hopefully it is now better.